From e097f20b0bf3ee64a75f72b46e7d3c351d3d28bd Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Thu, 18 Nov 2021 16:37:17 +0100
Subject: [PATCH] linux: enable CONFIG_BLK_DEV_NVME

We need to enable CONFIG_BLK_DEV_NVME.
It is a driver for for SSD connected to PCI or PCIe bus [1].

Reference:
[1] https://cateee.net/lkddb/web-lkddb/BLK_DEV_NVME.html

This patch can not be sent upstream, according to my information. 
They don't want to have enabled all the things by default. They would 
prefer to have a kernel module (kmod) to be installed.

Even though for some targets like x86, ipq807x, and rockchip it is
enabled by default.

That's why this patch is located in WIP folder in turris-build
repository.

---
 target/linux/generic/config-5.4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/target/linux/generic/config-5.4 b/target/linux/generic/config-5.4
index 680f0d0feb..8da9c12f67 100644
--- a/target/linux/generic/config-5.4
+++ b/target/linux/generic/config-5.4
@@ -589,7 +589,7 @@ CONFIG_BLK_DEV_LOOP_MIN_COUNT=8
 # CONFIG_BLK_DEV_NBD is not set
 # CONFIG_BLK_DEV_NS87415 is not set
 # CONFIG_BLK_DEV_NULL_BLK is not set
-# CONFIG_BLK_DEV_NVME is not set
+CONFIG_BLK_DEV_NVME=y
 # CONFIG_BLK_DEV_OFFBOARD is not set
 # CONFIG_BLK_DEV_OPTI621 is not set
 # CONFIG_BLK_DEV_PCIESSD_MTIP32XX is not set
-- 
2.30.2

